watchtower/wtpolicy/policy.go

```diff
@@ -0,0 +1,68 @@
+package wtpolicy
+
+import (
+	"fmt"
+
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/watchtower/blob"
+)
+
+const (
+	// DefaultMaxUpdates specifies the number of encrypted blobs a client
+	// can send to the tower in a single session.
+	DefaultMaxUpdates = 1024
+
+	// DefaultRewardRate specifies the fraction of the channel that the
+	// tower takes if it successfully sweeps a breach. The value is
+	// expressed in millionths of the channel capacity.
+	DefaultRewardRate = 10000
+
+	// DefaultSweepFeeRate specifies the fee rate used to construct justice
+	// transactions. The value is expressed in satoshis per kilo-weight.
+	DefaultSweepFeeRate = 3000
```

Comment:
Should there even be a default? Instead we can pass in a `lnwallet.FeeEstimator` instance and if we're on simnet or w/e, then that can be a static version of it.

Reply:
This is the fee rate that is negotiated between client and server. The client itself can have a fee estimator, and use that to prescribe what it attempts to negotiate. This is just a shared struct for holding the negotiated values.

Reply:
Sorry, ignore the prior comment, misread where the comment was.

There doesn't need to be a default, yes in practice you would use whatever the client fee estimator proposes. I've been using this mostly to generate valid policies for testing. Will remove it